public/crtlib.h

```diff
@@ -87,6 +88,7 @@ void COM_PathSlashFix( char *path );
 char COM_Hex2Char( uint8_t hex );
 void COM_Hex2String( uint8_t hex, char *str );
 #define COM_CheckString( string ) ( ( !string || !*string ) ? 0 : 1 )
+#define COM_CheckString2( string ) ( ( !*string ) ? 0 : 1 )
```

Comment:
I don't really like that we have COM_CheckString2 that just don't check for NULL pointer.

Can we just use COM_CheckString for preallocated strings? Are compilers are smart enough to always assume that they are never equal to NULL? I think this macro already used somewhere on preallocated string.

Reply:
Unfortunately, compiler gives warning if we check pointer for preallocated string.

Reply:
@nekonomicon then maybe give it more useful name? CheckStringEmpty? 

Just for reference, other languages standard libraries call this operation something like "isNullOrEmpty". We could adapt it, thus having COM_IsNullOrEmpty and COM_IsEmpty.

Reply:
Ok